specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/dedicatedHostExamples/DedicatedHost_Redeploy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-09-01",
+    "hostGroupName": "myDedicatedHostGroup",
+    "hostName": "myHost"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "location": "https://foo.com/operationstatus"
```

Comment:
>foo.com/ [](http://example.com/codeflow?start=29&length=8)

consider not using foo.com's domain

Reply:
we've been using it and a lot of compute's examples already contain them